src/js/common/schemaform/ObjectField.jsx

```diff
@@ -61,7 +63,14 @@ class ObjectField extends React.Component {
   }
 
   shouldComponentUpdate(nextProps, nextState) {
-    return shouldRender(this, nextProps, nextState);
+    const propsWithoutDataUnchanged = deepEquals(_.omit('formData', this.props), _.omit('formData', nextProps));
```

Comment:
This addresses some performance issues I ran into on older phones. We can skip updates when the state and props versions of formData are the same and everything else matches.